### ISSUE TITLE
job.tags() raises if tag is None

### ISSUE THREAD

**Issue by ajavadia** [Qiskit/qiskit-ibmq-provider#847]
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-207-85f0ddd9d598> in <module>
----> 1 job_xx_1.tags()

~/anaconda3/lib/python3.7/site-packages/qiskit/providers/ibmq/job/ibmqjob.py in tags(self)
    602             Tags assigned to this job.
    603         """
--> 604         return self._tags.copy()
    605 
    606     def time_per_step(self) -> Optional[Dict]:

AttributeError: 'NoneType' object has no attribute 'copy'

```

**Comment by jyu00** [comment]
@ajavadia Which version of `qiskit-ibmq-provider` were you using? This should have been fixed by #698.

**Comment by ajavadia** [comment]
ok i might have been out of date, can't find my original code. i'll close it, thanks for clarification.
